### ISSUE TITLE
Other taxonomic bases and indices

### ISSUE THREAD

**Issue by MaxCarabus** [PavelSyomin/checkplantlist#1]
Is it possible to handle other commonly knonw on-line botanical system? 

[An Online Flora of All Known Plants](http://www.worldfloraonline.org/)
[The World Checklist of Vascular Plants (WCVP)](https://wcvp.science.kew.org/)
[Plants of the World Online](http://www.plantsoftheworldonline.org/)
[International Plant Name Index (IPNI)](https://www.ipni.org/)
as well 
[GBIF Backbone Taxonomy](https://www.gbif.org/dataset/d7dddbf4-2cf0-4f39-9b2a-bb099caae36c)

**Comment by PavelSyomin** [comment]
Yes, and perhaps one day I will add them to the app :-) It's not a simple task I can complete in 30 minutes, and right now I don't have enough time to add new features to this project. But I agree that handling other databases is a great idea.
